### ISSUE TITLE
Export database statistic over API and CLI

### ISSUE THREAD

**Issue by gz-c** [skycoin/skycoin#1454]
After #1440 is merged,

add a method to collect database statistics (bolt's stats method, bucket lengths)

export this data over an API endpoint and add a CLI command to return it too

**Comment by stdevHan** [comment]
@gz-c I would like to know, an example of how the output would look.

**Comment by gz-c** [comment]
A JSON object that contains stats collected from the database

**Comment by stdevHan** [comment]
@gz-c I would like to know what statistics is exactly what is required to see the database

**Comment by gz-c** [comment]
whatever boltdb exposes in its stats method, and also the number of items in buckets if this is not in the stats

**Comment by gz-c** [comment]
@olemis can we tie this in to prometheus?

**Comment by olemis** [comment]
If exported through the REST API then `rextporter` can scrape it and convert into Prometheus format .
@stdevAlDen cmiiw

**Comment by olemis** [comment]
@stdevStark could u take a look into this ?

**Comment by stdevAlDen** [comment]
> @stdevStark could u take a look into this ?

sure, @stdevHan you can find a configuration example for skycoin node [here](https://github.com/simelo/rextporter/tree/develop/test/integration/skycoin/tomlconfig), some comments in the [readme](https://github.com/simelo/rextporter/blob/develop/README.md#config-file), let me kow if you need some other thing

**Comment by olemis** [comment]
@stdevAlDen is it complete ? Does it consider the new API's added in 0.25.1 ?

**Comment by stdevAlDen** [comment]
> @stdevAlDen is it complete ? Does it consider the new API's added in 0.25.1 ?

Yes, there do not seem to be many changes that affect metrics in [2.25.1](https://github.com/skycoin/skycoin/releases) but I have to prove it, @stdevHan this should not hinder your progress...
